examples/siro_sandbox/sandbox_app.py

```diff
@@ -1424,6 +1432,17 @@ def _parse_debug_third_person(args, framebuffer_size):
             )
             gui_controlled_agent_config.sim_sensors.clear()
 
+            for sensor_name in [
```

Comment:
What motivated this change? This looks like a great bugfix! Is this actually turning off some unnecessary rendering that we have been doing in the HITL tool?

Reply:
This is motivated by the fact that policy evaluation code expects to be operating on the the `GymHabitatEnv` (`habitat.Env` wrapped in a bunch of wrappers that modify/transform/filter observations returned by the env and also actions before they are passed to the env directly).

You may see that now I define two env references:
```python
self.gym_habitat_env: "GymHabitatEnv" = (
    habitat.gym.make_gym_from_config(config=config, dataset=dataset)
)
self.habitat_env: habitat.Env = (
    self.gym_habitat_env.unwrapped.habitat_env
)
```
GUI and app related core operates directly on `self.habitat_env` and baselines (agents) on self.gym_habitat_env.